MOxUnit/moxunit_util_elem2str.m

```diff
@@ -18,20 +18,54 @@
 %     to capture the output from disp in a variable; hence only the class
 %     of the first input is shown
 %
-% NNO Jan 2014
+% NNO, SCL 2014-2015
 
     if nargin<2
         max_chars=100;
     end
 
-    if moxunit_util_platform_is_octave()
-        elem_str=sprintf('<%s>\n',class(elem));
-    else
-        % Octave does not support evalc
+    % Strings are trivally supported by leaving them as-is
+    if ischar(elem)
+        % Encapsulate in quotation marks
+        elem_str=['''' elem ''''];
+
+    elseif isnumeric(elem)
+        % If the element is numeric, we can use mat2str
+        elem_str=mat2str(elem);
```

Comment:
Propose replacing with:
```
elseif isnumeric(elem) && numel(elem) < max_chars
```
And having a different handling for larger numerics, similar to the cell handler?